src/Account.py

```diff
@@ -12,7 +12,8 @@ def LoadEntity(db: AbstractDataAccess):
             ('act_lname', 'varchar(50) not null'),
             ('act_password', 'varchar(20)'),
             ('act_phone', 'varchar(12) unique not null'),
-            ('role_id', 'integer not null')
+            ('role_id', 'integer not null'),
+            ('act_address', 'varchar(70) not null')
```

Comment:
70 is probably fine, but just in case we could always bump this up to 255, that's just what I've seen from the databases we use at work. Not a huge deal, but just kind of as a precaution